spade-core/src/main/scala/com/salesforce/mce/spade/workflow/Resource.scala

```diff
@@ -14,5 +14,6 @@ case class Resource[+R](
   name: String,
   resourceType: String,
   resourceSpec: Json,
-  maxAttempt: Int
+  maxAttempt: Int,
+  terminateAfter: Option[Double]
```

Comment:
let's make this `Duration` and convert to `Double` when converting `workflow.Resource` to `WorkflowRequest.Resource`

Reply:
done